exonum-java-binding/core/rust/src/storage/fork.rs

```diff
@@ -20,6 +20,58 @@ use handle::{self, acquire_handle_ownership, Handle};
 use storage::db::View;
 use {to_handle, utils};
 
+/// Creates checkpoint for `Fork`.
+///
+/// Returns error if the provided handle is not owned `Fork`.
```

Comment:
it cannot return an error, it may only throw an exception

Reply:
Corrected